src/js/core.js

```diff
@@ -643,6 +596,54 @@ function MediumEditor(elements, options) {
         },
 
         /**
+         * NOT DOCUMENTED - exposed as a helper for other extensions to use
+         */
+        addBuiltInExtension: function (name) {
+            var extension = this.getExtensionByName(name);
+            if (extension) {
+                return extension;
+            }
+
+            switch(name) {
```

Comment:
I've updated our jscs config so now it complains about this as well.  There were only 2 issues and you managed to find both of them, so nice work :smile: